src/ServiceControl.Config/ServiceControl.Config.csproj

```diff
@@ -19,17 +19,22 @@
     <Reference Include="System.ServiceProcess" />
   </ItemGroup>
 
+  <!-- Fody and friends -->
+  <ItemGroup>
+    <PackageReference Include="Fody" Version="6.4.0" PrivateAssets="All" />
+    <PackageReference Include="PropertyChanged.Fody" Version="3.3.3" PrivateAssets="All" />
+    <PackageReference Include="PropertyChanging.Fody" Version="1.30.2" PrivateAssets="All" />
+    <PackageReference Include="Validar.Fody" Version="1.9.0" PrivateAssets="All" />
+  </ItemGroup>
+  
   <ItemGroup>
     <PackageReference Include="Autofac" Version="4.9.4" />
     <PackageReference Include="Caliburn.Micro" Version="3.2.0" />
     <PackageReference Include="FluentValidation" Version="9.5.3" />
-    <PackageReference Include="Fody" Version="6.4.0" />
     <PackageReference Include="Mindscape.Raygun4Net" Version="5.12.0" />
-    <PackageReference Include="PropertyChanged.Fody" Version="3.3.3" />
-    <PackageReference Include="PropertyChanging.Fody" Version="1.30.2" />
-    <PackageReference Include="reactiveui" Version="5.5.1" />
+    <PackageReference Include="reactiveui" Version="13.2.2" />
+    <PackageReference Include="ReactiveUI.WPF" Version="13.2.2" />
```

Comment:
This package is needed to be able to determine the main scheduler. Without it SC.Config crashes with IllegalCrossThreadException